backend/src/main/resources/application.properties

```diff
@@ -1,4 +1,5 @@
 quarkus.http.cors=true
+quarkus.http.cors.origins=http://localhost:8080
```

Comment:
Don't we have to configure this dynamically from the Docker containers?

Reply:
Removed